examples/diffusion_limited_aggregation.jl

```diff
@@ -0,0 +1,123 @@
+using Agents, Random, LinearAlgebra
+Random.seed!(42)
+
+@agent Particle ContinuousAgent{2} begin
+    radius::Float64
+    is_stuck::Bool
+    twist_axis::Array{Float64,1}
+end
+
+Particle(
+    id::Int,
+    radius::Float64,
+    twist_clockwise::Bool;
+    pos = (0.0, 0.0),
+    is_stuck = false,
+) = Particle(
+    id,
+    pos,
+    (0.0, 0.0),
+    radius,
+    is_stuck,
+    [0.0, 0.0, twist_clockwise ? -1.0 : 1.0],
+)
+
+properties = Dict(
```

Comment:
This is best to go in the initialization function, so that we can also add this to the `models` section.